pkg/cmd/roachtest/tests/network.go

```diff
@@ -350,9 +356,10 @@ sudo iptables -P INPUT ACCEPT;
 sudo iptables -P OUTPUT ACCEPT;
 
 # Drop any client traffic to CRDB.
-sudo iptables -A INPUT -p tcp --sport 26257 -j DROP;
-sudo iptables -A OUTPUT -p tcp --dport 26257 -j DROP;
-`
+sudo iptables -A INPUT -p tcp --sport {pgport%s} -j DROP;
```

Comment:
looks like Daryl also has this PR: https://github.com/cockroachdb/cockroach/pull/117909

but, i'm fine merging either one